docs/atomic-install.1.md

```diff
@@ -77,6 +77,10 @@ Installing a system container consists of checking it the image by
 default under /var/lib/containers/atomic/ and generating the
 configuration files for runc and systemd.
 
+**--user**
+If running as non-root, specify to install the image from the current
+OSTree repository and manage it through systemd and bubblewrap.
```

Comment:
Why not do the imply --user if running as non root?  Are you intent on treating run differently then install?